src/components/forms/event-form.js

```diff
@@ -977,19 +1027,35 @@ render() {
                 </div>
             }
             {this.shouldShowField('use_speakers') &&
-                <div className="row form-group">
-                    <div className="col-md-12">
-                        <label> {T.translate("general.speakers")} </label>
-                        <SpeakerInput
-                            id="speakers"
-                            value={entity.speakers}
-                            onChange={this.handleChange}
-                            multi={true}
-                            history={history}
-                            getOptionLabel={(speaker) => `${speaker.first_name} ${speaker.last_name} (${speaker.email})`}
-                        />
+                <>
+                    <div className="row form-group">
+                        <div className="col-md-10">
+                            <label> {T.translate("general.speakers")} </label>
+                            <SpeakerInput
+                                id="speaker"
+                                onChange={this.handleChange}
```

Comment:
on change should reset the current value

Reply:
fixed